model/riscv_platform.sail

```diff
@@ -26,6 +26,9 @@ val elf_entry = {
   c: "elf_entry"
 } : unit -> int
 
+// Cache block size is 2^cache_block_size_exp. Max 4096.
```

Comment:
Perhaps worth documenting why this is the implementation limit (minimum page size)? I don't see it required in the spec.

Reply:
Yeah it comes from `max_mem_access`, which I set to 4096 (it was 16 previously). Reading the spec it doesn't really mention the behaviour for cache blocks larger than a page (presumably because they thought that was not a sane configuration and didn't think about it), but I think it would bring up awkward questions if you tried to do that, so I just limited at 1 page.

I added a comment to hopefully explain that.